source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -247,35 +247,39 @@ namespace PlayFab
 
     void PlayFabAndroidHttpPlugin::MakePostRequest(std::unique_ptr<CallRequestContainerBase> requestContainer)
```

Comment:
Have you validated this does not break android C++ sdk ?

Reply:
I have. I kicked off a run of the XPlat on Jenkins a couple times, whenever I changed the non-windows files, and (Win/iOS/Android) have passed.
The consoles won't be affected until we apply these changes to those templates.

---
In reply to: [343404709](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343404709) [](ancestors = 343404709)